packages/teleport-monitoring/src/bin/utils.ts

```diff
@@ -17,9 +17,19 @@ type InitFunction = (args: {
   settleRepository: SettleRepository
 }) => Promise<void>
 
+interface Slave {
```

Comment:
Can we join the politically correct movement and move away from "slave"? :)

Reply:
Master / slave is a computer science term.

Reply:
Yeah let's not open this can of worms. I just follow what the geniuses at Github/Python/Golang/Chromium have decided :D Carry on